src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+  color: #3e4e61;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-family: "Avenir-book", sans-serif;
+}
```

Comment:
Will be better add default font to html tag and use needed font-weight for inputs and for ::placeholder